templates/rhsm-katello-reconfigure.erb

```diff
@@ -71,9 +71,14 @@ if [ -d $CA_TRUST_ANCHORS ]; then
   cp $KATELLO_CERT_DIR/$KATELLO_CERT $CA_TRUST_ANCHORS
   update-ca-trust
 
-  #restart if docker service is installed
-  service docker status >/dev/null && \
+  # restart docker if it is installed and running
+  if [ -f /usr/lib/systemd/system/docker.service ]; then
```

Comment:
@Ichimonji10 I tried your suggestion and get the following. Also I have to remove the docker.service option from `list-unit-files`. I will need to deal with the output of systemd not being available and that will make the code bigger. I will stick with 'Simple is better than complex.' here.

```
[root@rhel66 ~]# if systemctl list-unit-files | grep '^docker.service' >&/dev/null; then echo 'have docker'; else echo 'does not have docker'; fi
-bash: systemctl: command not found
does not have docker
```